### ISSUE TITLE
session_energy_wh in hass

### ISSUE THREAD

**Issue by donnib** [einarhauks/tesla-wall-connector#4]
Hi @einarhauks ,
Is there a reason for `session_energy_wh` beeing available in HASS since i can see it's in the python code here or am i missing something ?

/donnib

**Comment by einarhauks** [comment]
Did you mean to write: "Is there a reason for session_energy_wh NOT beeing available in HASS"?
This library exposes all data that one can retrieve from the lifetime, version and vitals endpoints of the Wall Connector. This is however a lot of data and only the most relevant data is made available in HASS. The main reason for not including the session energy was just to minimize confusion with the more useful lifetime energy, which can be used for the Energy Dashboard in HASS. Perhaps (current/last) session energy is interesting for someone, but in that case one can always make a pull request with that addition to the [Tesla Wall Connector HASS integration](https://github.com/home-assistant/core/tree/dev/homeassistant/components/tesla_wall_connector) :)

**Comment by donnib** [comment]
Yes sorry about the confusion, you got it right. So i guess using the lifetime energy one can see in the energy dashboard how much the charge was made during a specific day right ? That might suffice, i'll have to think about that.
